### ISSUE TITLE
Separate Analytics docs

### ISSUE THREAD

**Issue by Mariangela** [wso2/docs-apim#3982]
Description:
Review "Observe" section doc structure and create an "Analytics" section.

**Comment by Mariangela** [comment]
This task was on-hold until 21st May 2021 due to other urgent work that came up.

**Comment by Mariangela** [comment]
Had a general meeting on the overall analytics and observe section on the 13th May 2021 with Nuwan, Sam, Nilmini, Tishan, Fazlan and AmilaD

**Comment by Mariangela** [comment]
Had a meeting with Tishan and Dilini on the 25th May 2021 based on the MI and SI structures.

**Comment by Mariangela** [comment]
Had a meeting on the Analytics section with Nuwan and Tishan on 28th May 2021 (Friday) based on the documentation structure that I proposed via a gdoc.

**Comment by Mariangela** [comment]
Pushed changes that was discussed in the last meeting via PR [1].
Tasks done

- Added the API Analytics doc changes are under the new "Analytics" section.
- Getting Started guide has been constructed based on multiple pages to make it easy for the user to figure out the flow to get started.
- Redundant pages have been removed.
- API Gateway basic configs have been single sourced as it appears in two places.
- Left navigation has been updated based on the new convention (being less verbose) that you proposed.
- The added pages have been reviewed and updated.
- Images have been resized as requested.
- The architecture diagram, which was redrawn by the marketing team, has been added.
- Links have been updated.
- Broken links have been fixed.
- Redirections have been added.
- Ruthryi hid the Analytics related WIP pages and also updated the Choreo related screenshots which had black masking on it.
**To Do**
- The MI Analytics section is yet to be added.
This is blocked by Git issue [2]. @Nandika Jayawardana , can you please assign someone to address this issue.
- The Analytics dashboard pages in the docs have to be removed after the UI is reviewed for intuitivity.
This is blocked by Git issue [3]. @Amila De Silva , can you please assign someone to address this task.
- Choreo Connect configurations need to be added
These configurations have been mentioned separately in [4]. I need to have a discussion with Rajith and to see how best the content can be reflected under the "Analytics" section as well.
- The API Analytics Architecture [5] section needs to be improved.
 @Amila De Silva , please let me know your thoughts on this.

[1] https://github.com/wso2/docs-apim/pull/4128
[2] https://github.com/wso2/product-apim/issues/11290
[3] https://github.com/wso2/product-apim/issues/11289
[4] https://apim.docs.wso2.com/en/latest/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics/
[5] https://apim.docs.wso2.com/en/latest/api-analytics/architecture/

**Comment by Mariangela** [comment]
**On-Hold**
This task is on-hold as I have been assigned to work on [1]

[1] https://github.com/wso2/docs-apim/issues/4130

**Comment by Mariangela** [comment]
This task is further on-hold due to [2]

[2] https://github.com/wso2/docs-apim/issues/4157

**Comment by Mariangela** [comment]
This task is further on-hold due to [3]

[3] https://github.com/wso2/docs-apim/issues/4158

**Comment by Mariangela** [comment]
This task was further on-hold due to [4]

[4] https://github.com/wso2/docs-apim/issues/4164

**Comment by Mariangela** [comment]
This task was further on-hold due to [5]

[5] https://github.com/wso2/docs-apim/issues/4194

**Comment by Mariangela** [comment]
This task was further on-hold due to [6]

[6] https://github.com/wso2/docs-apim/issues/4203

**Comment by Mariangela** [comment]
On-hold for https://github.com/wso2/docs-apim/issues/4221

**Comment by Mariangela** [comment]
On hold for https://github.com/wso2/docs-apim/issues/4222

**Comment by Mariangela** [comment]
This is pending as the EI Analytics interface needs to be renamed to MI Analytics and pushed live via Updates 2.0.

**Comment by Mariangela** [comment]
Now that Damith has pushed the changes the MI content can be moved.

Email: Note for MI Analytics Portal change via Updates 2.0

**Comment by Mariangela** [comment]
On hold for https://github.com/wso2/docs-apim/issues/4279
